### PR TITLE
utopia prc vc v2

### PR DESCRIPTION
**Pull request by BigBlueHat**
- Add new PRC VC v2.
- Update CHANGELOG.

### CODE REVIEW

**Review by dlongley — APPROVED** [review]

File: credentials/prc-vc-v2/credential.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://w3id.org/citizenship/v4rc1"
+  ],
+  "id": "https://issuer.oidp.uscis.gov/credentials/83627465",
```
Comment (davidlehn) [inline credentials/prc-vc-v2/credential.json:6]: Use generic issuer URL.
```suggestion
  "id": "https://issuer.example.gov/credentials/83627465",
```

File: credentials/prc-vc-v2/queries.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "default": {
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your PermanentResidentCard or Verifiable Credential(s) to complete the verification process.",
+      "example": {
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1",
+          "https://w3id.org/citizenship/v1"
```
Comment (davidlehn) [inline credentials/prc-vc-v2/queries.json:9]:
```suggestion
          "https://www.w3.org/ns/credentials/v2",
          "https://w3id.org/citizenship/v4rc1"
```

File: credentials/prc-vc-v2/queries.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    }
+  },
+  "Permanent Resident (Country Only)": {
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your PermanentResidentCard or Verifiable Credential(s) to complete the verification process.",
+      "example": {
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1",
+          "https://w3id.org/citizenship/v1"
```
Comment (davidlehn) [inline credentials/prc-vc-v2/queries.json:29]:
```suggestion
          "https://www.w3.org/ns/credentials/v2",
          "https://w3id.org/citizenship/v4rc1"
```

File: credentials/prc-vc-v2/queries.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  "default": {
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your PermanentResidentCard or Verifiable Credential(s) to complete the verification process.",
+      "example": {
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1",
+          "https://w3id.org/citizenship/v1"
+        ],
+        "type": ["PermanentResidentCard"]
```
Comment (davidlehn) [inline credentials/prc-vc-v2/queries.json:11]:
```suggestion
        "type": ["PermanentResidentCardCredential"]
```

File: credentials/prc-vc-v2/queries.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  "Permanent Resident (Country Only)": {
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your PermanentResidentCard or Verifiable Credential(s) to complete the verification process.",
+      "example": {
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1",
+          "https://w3id.org/citizenship/v1"
+        ],
+        "type": ["PermanentResidentCard"],
```
Comment (davidlehn) [inline credentials/prc-vc-v2/queries.json:31]:
```suggestion
        "type": ["PermanentResidentCardCredential"]
```

**Pull request merged**